policy_definitions/deploy_log_analytics_agent/policy_definition_qby_deploy_log_analytics_agent.json

```diff
@@ -0,0 +1,139 @@
+{
+    "name": "Deploy-Log-Analytics-Agent-to-Windows-VM",
+    "type": "Microsoft.Authorization/policyDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "mode": "Indexed",
+        "displayName": "Deploy - Configure Log Analytics extension to be enabled on Windows virtual machines",
+        "description": "Deploy Log Analytics extension for Windows virtual machines if the virtual machine image is in the list defined and the extension is not installed.",
+        "metadata": {
+            "category": "Guest Configuration",
+        },
+        "parameters": {
+            "listOfImageSKUs": {
+                "type": "Array",
+                "metadata": {
+                    "description": "The list of image SKUs where the Log Analytics Agent will be installed.",
+                    "displayName": "List of Image SKUs"
+                }
+            },
+            "logAnalytics": {
+                "type": "String",
+                "metadata": {
+                    "displayName": "Log Analytics workspace",
+                    "description": "Resource Id of the Log Analytics workspace used to receive performance data.",
+                    "assignPermissions": true
+                }
+            }
+        },
+        "policyRule": {
+            "if": {
+                "allOf": [
+                    {
+                        "field": "type",
+                        "equals": "Microsoft.Compute/virtualMachines"
+                    },
+                    {
+                        "field": "Microsoft.Compute/imagePublisher",
+                        "equals": "MicrosoftWindowsServer"
+                    },
+                    {
+                        "field": "Microsoft.Compute/imageSKU",
+                        "in": "[parameters('listOfImageSKUs')]"
+                    }
```

Comment:
Do we want to use a matching instead, to avoid specifying every possible version?